src/display/svg.js

```diff
@@ -936,6 +946,91 @@ SVGGraphics = (function SVGGraphicsClosure() {
       this.current.tspan = this.svgFactory.createElement('svg:tspan');
       this.current.xcoords = [];
     },
+    setStrokeColorN: function SVGGraphics_setStrokeColorN(args) {
+      this.endPathcurrent.strokeColor = this._makeColorN_Pattern(args);
```

Comment:
Shouldn't this be `this.current.strokeColor`?

Reply:
Thank you! I fixed this mistake.